### ISSUE TITLE
Improve Verifying Signatures for Windows and Mac users

### ISSUE THREAD

**Issue by andrewdavidwong** [QubesOS/qubes-issues#6191]
**The problem you're addressing (if any)**

[Verifying Signatures](https://www.qubes-os.org/security/verifying-signatures/) was written primarily with Linux in mind. It needs to be improved for Windows and Mac users.

**Describe the solution you'd like**

Add or enhance instructions for Windows and Mac users.

**Where is the value to a user, and who might that user be?**

People coming from Windows and Mac will be able to verify the Qubes ISO on their current systems.

**Describe alternatives you've considered**

Pointing to external documentation. It'd be nicer if we could provide exact commands that will work.

**Additional context**

https://github.com/QubesOS/qubes-doc/pull/1076

https://qubes-os.discourse.group/t/iso-verification-instructions-cli-commands-arent-recognized-by-my-non-qubes-cli/1373

**Relevant [documentation](https://www.qubes-os.org/doc/) you've consulted**

https://www.qubes-os.org/security/verifying-signatures/

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**

None found.

**Comment by andrewdavidwong** [comment]
I've added a "Preparation" section that points out programs for Windows and Mac users. Unfortunately, however, I don't have access to such systems for testing commands writing specific documentation for those OSes.

@PROTechThor, any chance you might want to do this?

**Comment by PROTechThor** [comment]
> I've added a "Preparation" section that points out programs for Windows and Mac users. Unfortunately, however, I don't have access to such systems for testing commands writing specific documentation for those OSes.
> 
> @PROTechThor, any chance you might want to do this?

@andrewdavidwong Sorry, I can't help here. I don't have those systems either.

**Comment by DemiMarie** [comment]
One idea would be to write a download tool and sign it with an ITL code-signing certificate.  That avoids users needing to worry about verifying the keys themselves, since the tool would be signed with a certificate trusted by Windows.

**Comment by deeplow** [comment]
> One idea would be to write a download tool and sign it with an ITL code-signing certificate. That avoids users needing to worry about verifying the keys themselves, since the tool would be signed with a certificate trusted by Windows.

Good point. If I were windows user (unfamiliar with GPG) verification would probably be very daunting and there would be some steps that I wouldn't probably be able to make securely  (Even a lot of qubes users [don't do this properly](https://qubes-os.discourse.group/t/signed-q4-1-alpha-iso/981/6)).

Tails has overcome this burden similarly to your suggestion @DemiMarie instead making a Firefox extension: the [Tails Verfication](https://tails.boum.org/contribute/design/verification_extension/). So they instead delegate the trust to mozilla. Some of their reasons:

> * Using HTTPS to download. But in the case of Tails, we are serving so many downloads that we have to rely on mirrors hosted by third parties. HTTPS also doesn't protect from interrupted downloads leading to broken Tails installations.
>
> * Providing OpenPGP signatures. But this really works only for the few people who know how to verify an OpenPGP signature and use the OpenPGP Web-of-Trust correctly.

Something else they don't mention but is equally valid is that by making a browser extension, it works for both mac, windows and linux.

**Comment by hackerncoder** [comment]
I have access to a Windows computer, however no mac. I can do some of it.

&nbsp;

> a browser extension

I am all for this. It is an easy solution for users, that doesn't require them to install software on their computers (how does mac install software?).

**Comment by andrewdavidwong** [comment]
Thanks to Qubes Forum user **catacombs** for pointing out that Tails has replaced their browser extension verification method with JavaScript that runs on the download page:

https://tails.boum.org/news/verification_extension_deprecation/index.en.html

https://tails.boum.org/contribute/design/download_verification/

https://qubes-os.discourse.group/t/news-of-download-verification-on-tails-linux/1874

**Comment by DemiMarie** [comment]
:disappointed:

**Comment by TranceV4** [comment]
Jesus… just decided to impossible to get this qubes for who have such windows that’s makes me feel an *another* one thanks!

**Comment by DemiMarie** [comment]
My recommendation is to offer a USB creator signed by proper Windows and macOS code-signing certificates.
